code/modules/recycling/conveyor2.dm

```diff
@@ -246,9 +281,6 @@
 			S.update()
 		CHECK_TICK
 

```

Comment:
The switches might do well to store a list of one another, if they're bothering to save a list of conveyors on the same network at time of switch initialization